### ISSUE TITLE
Sending player where they're not whitelisted crashes the proxy

### ISSUE THREAD

**Issue by JosTheDude** [smuddgge/Leaf#14]
`Leaf Version:`
Latest (1.4.1)

**Description of the Bug**
When you send a player to a server in which they are not whitelisted, the Velocity proxy crashed

**Console Errors**
```
No errors returned
```

**Comment by smuddgge** [comment]
Hey, thanks for reporting. Will look into this :3

**Comment by smuddgge** [comment]
What version is your velocity proxy?

**Comment by JosTheDude** [comment]
```[14:51:42 INFO]: Velocity 3.2.0-SNAPSHOT (git-8761d02d-b225)
[14:51:42 INFO]: Copyright 2018-2021 Velocity Contributors. Velocity is licensed under the terms of the GNU General Public License v3.
[14:51:42 INFO]: velocitypowered.com - GitHub```

**Comment by smuddgge** [comment]
This should be fixed with `version 2.0.0`
